### PR TITLE
docs: add tile with tag pattern

### LINKED ISSUE #2006: [feat] Add `<rh-tag>` to `<rh-tile>`
### Description

Request from Dan for home page work.

Need a slot in `<rh-tile>` to add `<rh-tag>`.

### Suggested solution

See above.

### Screenshots

https://www.figma.com/proto/cB8q15K9rR3UgqQRM4CZzV/redhat.com-home-page---Sub-hero-section?page-id=1237%3A8801&node-id=1373-2368&node-type=section&viewport=-2130%2C-103%2C0.13&t=FNJRHzrQb4NWCMwJ-1&scaling=min-zoom&content-scaling=fixed

### Example API

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Added tile with tag pattern

Closes #2006 

## Testing Instructions

1. View the [Deploy Preview](https://deploy-preview-2008--red-hat-design-system.netlify.app/patterns/tile/examples/)

### CODE REVIEW

**Comment by coreyvickery** [comment]
@zeroedin Based on this, can we add 16px of space below the tag?

https://codepen.io/markcaron/pen/wvVpKVy?editors=1100

File: docs/patterns/tile/patterns/accented-tile.html [hunk 1]
```diff
@@ -4,6 +4,8 @@ <h2 slot="headline"><a href="#top">Link</a></h2>
 </rh-tile>
 
 <style>
+@import url('/assets/packages/@rhds/elements/elements/rh-tile/rh-tile-lightdom.css');
```
Comment (bennypowers) [inline docs/patterns/tile/patterns/accented-tile.html:7]: pattern files shouldn't have any uxdot-specific stuff (i.e. `/assets/packages/@rhds/elements/elements/...`) in them

I realize there might not be a good workaround in this case. We'll be able to solve it once-and-for all in #1801

**Review by bennypowers — APPROVED** [review]
Lovely green tags, mate

**Comment by coreyvickery** [comment]
@OneEightyFirst You can use this pattern on the redhat.com home page for the new sub-hero section.

**Comment by zeroedin** [comment]
> @zeroedin Based on this, can we add 16px of space below the tag?

Fixed 
https://deploy-preview-2008--red-hat-design-system.netlify.app/patterns/tile/examples/

File: docs/patterns/tile/patterns/accented-tile.html (hunk 1 above)
Comment (bennypowers) [reply docs/patterns/tile/patterns/accented-tile.html:7]: It's fine

**Pull request merged**